src/libraries/token/ERC20Basic.sol

```diff
@@ -7,8 +7,14 @@ pragma solidity ^0.4.13;
  * @dev see https://github.com/ethereum/EIPs/issues/179
  */
 contract ERC20Basic {
-    uint256 public totalSupply;
-    function balanceOf(address who) constant returns (uint256);
-    function transfer(address to, uint256 value) returns (bool);
+    uint256 internal totalSupply;
+
     event Transfer(address indexed from, address indexed to, uint256 value);
+
+    function getTotalSupply() public constant returns (uint256) {
```

Comment:
This function needs to remain named `totalSupply` so it is compatible with ERC20.  To achieve this, I recommend removing the `totalSupply` variable from this contract and implementing `function totalSupply(...)` in `StandardToken` using a backing variable named `supply` (to avoid name conflict with the function).  Mark it as `internal`, and you'll need to also edit `VariableSupplyToken.sol` to use the new internal variable name.

Reply:
Ok, sounds good.  I'll make this change.